sim/druid/talents.go

```diff
@@ -265,7 +265,7 @@ func (druid *Druid) applyOmenOfClarity() {
 				return
 			}
 
-			if spell.Flags.Matches(SpellFlagOmen) {
+			if spell.Flags.Matches(SpellFlagOmen) && spell.DefaultCast.Cost > 0 {
```

Comment:
`spell.CurCast.Cost`

Reply:
Oh dang you're right

Reply:
Actually no because omen makes the spell not cost anything but this specifically covers a hotfix where free wraths from the Fury of Stormrage elune do not consume clearcasting now